src/main/scala/mesosphere/chaos/http/HttpModule.scala

```diff
@@ -40,6 +42,20 @@ class HttpModule(conf: HttpConf) extends AbstractModule {
     } yield connector
   }
 
+  private def configureConnectorAddress(connector: Connector, addressOpt: ScallopOption[String], portOpt: ScallopOption[Int]): Unit = {
```

Comment:
Ok. Actually, many other methods should be private here. Then I change that as well.